lib/internal/Magento/Framework/Backup/Filesystem/Iterator/Filter.php

```diff
@@ -3,35 +3,34 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+namespace Magento\Framework\Backup\Filesystem\Iterator;
+
+use Iterator;
 
 /**
  * Filter \Iterator
  *
  * @author      Magento Core Team <core@magentocommerce.com>
  */
-namespace Magento\Framework\Backup\Filesystem\Iterator;
-
-use Iterator;
-
 class Filter extends \FilterIterator
 {
     /**
      * Array that is used for filtering
      *
      * @var array
      */
-    protected $_filters;
+    protected $filters;
```

Comment:
Please revert this change as it's not backward compatible. More info https://devdocs.magento.com/contributor-guide/backward-compatible-development/